test/test_tracker.cpp

```diff
@@ -565,6 +566,162 @@ TORRENT_TEST(current_tracker)
 	std::printf("done\n");
 }
 
+#if TORRENT_USE_RTC
+TORRENT_TEST(parse_websocket_tracker_response)
+{
+	char const response[] = "{\"complete\":1,\"incomplete\":0,\"action\":\"announce\",\"interval\":120,\"info_hash\":\"xxxxxxxxxxxxxxxxxxxx\"}";
```

Comment:
these example JSON response strings may be good candidates to use raw string literals for. i.e.
```
char const response[]
   = R"({"complete":1,"incomplete":0,"action":"announce","interval":120,"info_hash":"xxxxxxxxxxxxxxxxxxxx"})";
```